libs/langchain-google-common/package.json

```diff
@@ -39,7 +39,9 @@
   "author": "LangChain",
   "license": "MIT",
   "dependencies": {
-    "@langchain/core": "~0.1.1"
+    "@langchain/core": "~0.1.1",
+    "zod": "^3.22.4",
```

Comment:
Zod is only used in a `test.ts` file. Should be inside `devDependencies` instead

Reply:
Huh! So it is.